services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainMetrics.java

```diff
@@ -104,6 +108,11 @@ public BeaconChainMetrics(
         "previous_justified_root",
         "Current previously justified root",
         this::getPreviousJustifiedRootValue);
+    metricsSystem.createGauge(
+        TekuMetricCategory.BEACON,
+        "peer_count",
+        "Tracks number of connected peers",
```

Comment:
```suggestion
        "Tracks number of connected peers, verified to be on the same chain",
```

We may need to update the description for the libp2p_peer_count metric as well to make it clear that it's counting connections - otherwise people will just be confused why we have two metrics for the same thing with different values.